app/mailers/user_mailer.rb

```diff
@@ -9,21 +9,26 @@ class UserMailer < ActionMailer::Base
   end
 
   # checks the status of the current reservation and sends the appropriate email
-  # receipt forces a check out receipt to be sent
-  def reservation_status_update(reservation, receipt = false) # rubocop:disable all
+  # force forces a check out receipt to be sent OR an approved request email,
+  # (depending on the reservation status)
+  def reservation_status_update(reservation, force = false) # rubocop:disable all
     set_app_config
 
     @reservation = reservation
     @status = @reservation.human_status
 
-    return if !receipt && @status == 'returned' && @reservation.overdue &&
+    return if !force && @status == 'returned' && @reservation.overdue &&
               @reservation.equipment_model.late_fee == 0
 
-    return if receipt && @reservation.checked_out.nil?
+    return if force &&
+              (@reservation.checked_out.nil? && !@reservation.approved?)
```

Comment:
No, because then it doesn't send emails for checkout receipts that are not for reservations that were once requests, nor for approved requests that are not checked out. It is a bit messy to read though, I'll see if I can make it better.